README.md

```diff
@@ -1,19 +1,98 @@
-### Nightly Build Status (develop branch)
+# Hydroshare _(hydroshare)_
 
```

Comment:
It's just part of the "Standard Readme" format, something called the "package" name, or the machine-readable name. I guess psychologically it's also sort of softly claiming the name _hydroshare_ to belong to us in the software world.